remote-query/remote-query-client/pom.xml

```diff
@@ -19,11 +19,6 @@
          <artifactId>infinispan-commons</artifactId>
       </dependency>
 
-      <dependency>
-         <groupId>org.infinispan.protostream</groupId>
```

Comment:
yes, we get this dependency anyway transitively from commons. but why remove it? it is also useful to document a fact